guides/common/modules/proc_removing-packages-from-a-host.adoc

```diff
@@ -5,10 +5,11 @@ You can remove packages from a host in the {ProjectWebUI}.
 
 .Procedure
 . In the {ProjectWebUI}, navigate to *Hosts* > *All Hosts*.
-. Click the name of the host you want to modify.
-. Click the *Content* tab, then click the *Packages* tab.
-. Click the vertical ellipsis for the package you want to remove and select *Remove*.
-You get a REX job notification once the remote execution job is complete.
+. Select a host.
+. On the *Content* tab, click the *Packages* tab.
```

Comment:
This is outside the scope of your PR. I am auditing this guide and noticed that this assembly is very old. 

I logged in to Foreman and went to Hosts > All Hosts. Then I clicked a host to modify.

No Content tab, no Packages tab. I will open a bug.

Reply:
Aha. Just found a RHEL host with a Content tab!